apps/website/src/pages/docs/tabs.mdx

```diff
@@ -9,27 +9,126 @@ group: core
+
+```tsx
+<Tabs.Wrapper>
+  <Tabs.TabList>
+    <Tabs.Tab value='apple' label='Apple' key='apple' />
+    <Tabs.Tab value='orange' label='Orange' key='orange' />
+    <Tabs.Tab value='pear' label='Pear' key='pear' />
+  </Tabs.TabList>
+
+  <Tabs.Panel value='apple' key='apple'>
+    {/*...Content for apple*/}
+  </Tabs.Panel>
+  <Tabs.Panel value='orange' key='orange'>
+    {/*...Content for orange*/}
+  </Tabs.Panel>
+  <Tabs.Panel value='pear' key='pear'>
+    {/*...Content for pear*/}
+  </Tabs.Panel>
+</Tabs.Wrapper>
+```
+
+`Tabs.Tab` and `Tabs.Panel` can be associated with each other by passing them the same `value`.
+
+`Tabs.Tab` has `label` prop for ease of use. You can also build `Tabs.Tab` with subcomponents and add sublabel and icon to your tab.
+
+```tsx
+<Tabs.Tab value='apple' key='apple' label='Apple' />
+```
+
+```tsx
+<Tabs.Tab value='apple' key='apple'>
+  <Tabs.TabIcon>
+    <SvgAppleIcon />
+  </Tabs.TabIcon>
+  <Tabs.TabLabel>Apple</Tabs.TabLabel>
+  <Tabs.TabDescription>A red fruit</Tabs.TabDescription>
+</Tabs.Tab>
+```
+
+### Props API
+
+Alternatively, you can use Tabs directly with labels, actions, and children as panels. Although this pattern can suffice for simple cases, we strongly recommend using composition API for the best experience.
+
+The following code is equivalent to the composition example above:
+
+```tsx
+<Tabs
+  labels={[
+    <Tab key='apple' label='Apple' />,
+    <Tab key='orange' label='Orange' />,
+    <Tab key='pear' label='Pear' />,
+  ]}
+>
+  {/* … content for active tab */}
+</Tabs>
+```
+
+### Controlled state
+
+You can use `value` and `onValueChange` in `Tabs.Wrapper` to have fully controlled state. You are manually controlling active tab in the example bellow.
+
+You can also change tab that is active on component render using `defaultValue` prop.
+
+<LiveExample src='Tabs.controlled.tsx'>
+  <AllExamples.TabsControlledExample client:load />
+</LiveExample>
+
+## Variants
+
+### Vertical
+
+<LiveExample src='Tabs.vertical.tsx'>
+  <AllExamples.TabsVerticalExample client:load />
+</LiveExample>
+
+### Borderless
+
+As their name implies, borderless tabs have no frame or stroke surrounding them.
+
+Use borderless tabs when the tabbed content can be easily told apart from its surroundings (or lack thereof).
+
+<LiveExample src='Tabs.borderless.tsx'>
+  <AllExamples.TabsBorderlessExample client:load />
+</LiveExample>
+
+### Pill
+
+Pill tabs are meant to be used with content that belong to a similar category. They act like a filter for toggling through different states of similar content.
+
+The maximum recommended amount of pill tabs is 4.
+
+<LiveExample src='Tabs.pill.tsx'>
+  <AllExamples.TabsPillExample client:load />
+</LiveExample>
+
+### Actions
+
+You can add action buttons to the end of tab list.
+
+<LiveExample src='Tabs.actions.tsx'>
+  <AllExamples.TabsActionsExample client:load />
+</LiveExample>
+
 ### Overflow
```

Comment:
we could remove the overflow section as it does not need any code changes

Reply:
Removed